### DISCUSSION TITLE
如何隐藏目录里的文件夹

### DISCUSSION THREAD

**Discussion by LovelyTitan** [AlistGo/alist#2066] [Q&A]
alist版本3.2.0
目录里有两个文件夹：#recycle，@eaDir
想在全局隐藏以上两个文件夹，设置里看到可能的相关选项，但是各种修改都无法实现隐藏。请问该在哪里如何设置？

**Comment by nitsort** [comment]
https://alist.nn.ci/zh/config/global.html#%E9%9A%90%E8%97%8F%E6%96%87%E4%BB%B6
https://alist.nn.ci/zh/guide/advanced/meta.html#%E9%9A%90%E8%97%8F

**Comment by iucvcom** [comment]
可以在元信息的隐藏里面使用以下正则表达式
^\#recycle$
^\@recycle$

**Reply by CrazyJasonwell** [reply]
我按这个加了，好像没有作用

**Reply by cpt-kenvie** [reply]
^recycle$
recycle改成文件夹名字
